javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,13 @@
+//Goodboy-Oldboy
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+var dogYear = dogYearFuture - dogYearOfBirth;
```

Comment:
You rarely see developers use `var`. 
These days, you either use 
- `let` (for values that will be reassigned) or
- `const` (for values that won't be reassigned)

In this case `dogYear` is reassigned.

Reply:
I used let for dogYear now